disabled-packages/help2man/build.sh

```diff
@@ -7,3 +7,9 @@ TERMUX_PKG_VERSION=1.47.5
 TERMUX_PKG_SHA256=7ca60b2519fdbe97f463fe2df66a6188d18b514bfd44127d985f0234ee2461b1
 TERMUX_PKG_SRCURL=http://mirrors.kernel.org/gnu/help2man/help2man-${TERMUX_PKG_VERSION}.tar.xz
 TERMUX_PKG_BUILD_IN_SRC=yes
+
+Termux_step_make_install () {
```

Comment:
This should read `termux_step_make_install`, the first character being lowercase. Otherwise this is just a no-op, as build-package.sh won't call it.

Reply:
Thanks for pointing out